src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -41,6 +41,21 @@ For Azure (`--provider azure`):
 - [`AzureMachine`]({{< relref "/ui-api/management-api/crd/azuremachines.infrastructure.cluster.x-k8s.io.md" >}}) (API version `infrastructure.cluster.x-k8s.io/v1alpha3`) - specifies the control plane nodes.
 
 {{< /tab >}}
+{{< tab id="flags-capz" title="CAPI on Azure">}}
```

Comment:
I would prefer `Cluster API` over `CAPI`

Reply:
Works for me. Changed.